lib/MyChartAPI.js

```diff
@@ -138,12 +174,12 @@ async function AddFutureWeeks(
     weekCount,
     postDataCallback
 ) {
-    const results = { availability: {}, hasAvailability: false };
+    var results = {};
```

Comment:
We shouldn't be using `var` anywhere in this project - instead we'll use `const` or `let`. If you have a variable that needs reassignment, you'll use `let`; otherwise, `const`. Gotcha: objects can have their keys and values modified while still being `const`s (because of pointers and all that jazz).